app/models/miq_alert_status.rb

```diff
@@ -1,4 +1,36 @@
 class MiqAlertStatus < ApplicationRecord
   belongs_to :miq_alert
   belongs_to :resource, :polymorphic => true
+  has_many :miq_alert_status_states
+  SEVERITY_LEVELS = %w(danger warning info).freeze
```

Comment:
no, @serenamarie125 those were the three level we agreed about? or you rather have different names?